### PR TITLE
Compsupp 7274 - Fix fatal error when Elementor settings from widget are empty

### PR DESCRIPTION
**Pull request by diiegopereira**
See compsupp-7274

### CODE REVIEW

File: inc/class-mltools-elementor-config-generator.php [hunk 1]
```diff
@@ -60,9 +60,14 @@ public function get_widgets_list( $elements ) {
         $widgets = array();
 
         foreach ( $elements as $element ) {
-            if ( $element->elType === 'widget' && isset( $element->settings ) && is_object( $element->settings ) )  {
-                $widgetType                    = $element->widgetType;
-                $settings                      = (array) get_object_vars( $element->settings );
+            if ( $element->elType === 'widget' && isset( $element->settings ) && is_object( $element->settings ) ) {
+                $widgetType                   = $element->widgetType;
```
Comment (dgwatkins) [inline inc/class-mltools-elementor-config-generator.php]:
```suggestion
                $widgetType = $element->widgetType;
```

File: inc/class-mltools-elementor-config-generator.php [hunk 2]
```diff
@@ -60,9 +60,14 @@ public function get_widgets_list( $elements ) {
...
+
+                $settings = $element->settings;
+
+                if ( is_object( $settings ) ) {
+                    $settings                 = (array) get_object_vars( $settings );
```
Comment (dgwatkins) [inline inc/class-mltools-elementor-config-generator.php]:
```suggestion
                    $settings = (array) get_object_vars( $settings );
```

**Review by dgwatkins — CHANGES_REQUESTED** [review]
@diiegopereira It looks good, I was only able to suggest a couple of alignment suggestions. If you apply them I will merge and release.

**Comment by diiegopereira** [comment]
@dgwatkins thanks! I just applied them :)

File: multilingual-tools.php [hunk 3]
```diff
@@ -5,7 +5,7 @@
 Description: Set of tools to test themes and plugins multilingual compatibility.
 Author: OnTheGoSystems
 Author URI: https://www.onthegosystems.com/
-Version: 2.2.4
+Version: 2.2.5
 */
 
 define('WPML_CTT_VERSION', '2.2.4');
```
Comment (dgwatkins) [inline multilingual-tools.php]: Oops @diiegopereira 

I just noticed the version number missing here:
```suggestion
define('WPML_CTT_VERSION', '2.2.5');
```

I will go ahead and apply it myself.

**Review by dgwatkins — APPROVED** [review]

**Pull request merged**